libcerf/plan.sh

```diff
@@ -1,10 +1,10 @@
 pkg_name=libcerf
 pkg_origin=core
-pkg_version="1.5"
+pkg_version="1.13"
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=('MIT')
-pkg_source="http://apps.jcns.fz-juelich.de/src/$pkg_name/old/$pkg_name-$pkg_version.tgz"
-pkg_shasum="e36dc147e7fff81143074a21550c259b5aac1b99fc314fc0ae33294231ca5c86"
+pkg_source="https://jugit.fz-juelich.de/mlz/$pkg_name/uploads/924b8d245ad3461107ec630734dfc781/$pkg_name-$pkg_version.tgz"
```

Comment:
Is the hash in this URL meaningful or is it something Gitlab uses in its upload file schema?
If the hash is effectively random for us, it seems a little odd (imo) to interpolate the plan variables in the URL.

Reply:
I honestly don't know the answer to that. They've only done one release since moving to gitlab so I didn't check. If I had to guess, I'd say it's an md5 of _something_ so will likely change.